### PR TITLE
Fix Symmetry/Euler marker preprocessing logic with deforming markers

### PR DESCRIPTION
**Pull request by pcarruscag**
Fixes #1931

### CODE REVIEW

**Review by bigfooted — APPROVED** [review]
Great work, thanks!

**Review by WallyMaier — APPROVED** [review]
LGTM

File: TestCases/parallel_regression.py [hunk 1]
```diff
@@ -1337,6 +1337,16 @@ def main():
     pywrapper_rigidMotion.unsteady      = True
     test_list.append(pywrapper_rigidMotion)
 
+    # Deforming Bump in Channel
+    pywrapper_deformingBump = TestCase('pywrapper deforming bump in channel')
+    pywrapper_deformingBump.cfg_dir = "py_wrapper/deforming_bump_in_channel"
+    pywrapper_deformingBump.cfg_file = "config.cfg"
+    pywrapper_deformingBump.test_iter = 2
+    pywrapper_deformingBump.test_vals = [0.5, 0, -2.55436, -1.084594, -0.024882, 2.907803, 8.785498, -0.363585]
```
Comment (pcarruscag) [inline TestCases/parallel_regression.py:1345]:
```suggestion
    pywrapper_deformingBump = TestCase('pywrapper_deformingBump')
    pywrapper_deformingBump.cfg_dir = "py_wrapper/deforming_bump_in_channel"
    pywrapper_deformingBump.cfg_file = "config.cfg"
    pywrapper_deformingBump.test_iter = 1
    pywrapper_deformingBump.test_vals = [0.5, 0, -2.55436, -1.084594, -0.024882, 2.907803, 8.785498, -0.363585]
```

File: TestCases/py_wrapper/deforming_bump_in_channel/run.py [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+
+    # Update control parameters
+    TimeIter += 1
+    time += deltaT
+
+  # Postprocess the solver and exit cleanly
+  SU2Driver.Postprocessing()
+
+  if SU2Driver != None:
+    del SU2Driver
```
Comment (pcarruscag) [inline TestCases/py_wrapper/deforming_bump_in_channel/run.py:96]:
```suggestion
```

File: TestCases/py_wrapper/deforming_bump_in_channel/run.py [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+from mpi4py import MPI
+import numpy as np
+
+def main():
+  comm = MPI.COMM_WORLD
+  rank = comm.Get_rank()
+
+  # Initialize the corresponding driver of SU2, this includes solver preprocessing.
+  try:
+    SU2Driver = pysu2.CSinglezoneDriver('config.cfg', 1, comm);
```
Comment (pcarruscag) [inline TestCases/py_wrapper/deforming_bump_in_channel/run.py:37]:
```suggestion
    SU2Driver = pysu2.CSinglezoneDriver('config.cfg', 1, comm)
```

**Pull request merged**